### PR TITLE
[JS->TS] Migrate simple scripts with "few" imports

### PR DESCRIPTION
**Pull request by bh2smith**
This PR is intended to migrate as many `scripts/*.js` files as possible without touching the `utilities.js`. I think the following 4 files fit well together for this PR:

- [x] `cancel_orders.ts`
- [x] `close_auction.ts` --> ⚠️ this could probably be removed as it can only be used with ganache and was really only exposed for bash related e2e testing. All opposed?
- [x] `request_withdraw.ts`
- [x] `claim_withdraw.ts`

Partially fulfils #330

**Test Plan**

For Order Cancelation:

```sh
npx truffle exec build/common/scripts/cancel_order.js --network rinkeby --accountId 0 --orderIds 1,2,3
```

For Closing Auction:

```sh
npx truffle exec build/common/scripts/close_auction.js
```

Request and Claim Withdraw:

```sh
npx truffle exec build/common/scripts/request_withdraw.js --network rinkeby --accountId 0 --tokenId 1 --amount 1
```

```sh
npx truffle exec build/common/scripts/request_withdraw.js --network rinkeby --accountId 0 --tokenId 1
```

### CODE REVIEW

File: scripts/cancel_order.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { factory } from "../ConfigLog4";
```
Comment (nlordell) [inline scripts/cancel_order.ts:1]: ubernit: I noticed this file name after the PR had already been merged, I'm not a huge fan of it. I would rather it be `src/logging.ts` and not in the root directory.

**Review by nlordell — APPROVED** [review]
LGTM!

**Review by fleupold — APPROVED** [review]
Even if the scripts are small, I think it would be nice to have one PR per script...

File: scripts/cancel_order.ts (hunk 1 above)
Comment (bh2smith) [reply scripts/cancel_order.ts:1]: Yea cool! I am still unfamiliar with the file structure of a TS project and will change it here. I assume this is not meant to be exported.

**Comment by bh2smith** [comment]
> one PR per script

No problem, I will change this now.

**Comment by bh2smith** [comment]
Closing in favor of #804, #805, #806, #807 and #808. Note that all are based on #804 which implements the great suggestion to move the log configuration file.

**Pull request closed without merging**